src/screens/CompareModels/CompareModels.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import LazyLoad from 'react-lazyload';
+import FormControl from '@material-ui/core/FormControl';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+
+import ModelChart from 'components/Charts/ModelChart';
+import { STATES, STATE_TO_INTERVENTION } from 'enums';
+import { useModelDatas } from 'utils/model';
+import { buildInterventionMap } from 'screens/ModelPage/ModelPage';
+
+import {
+  Wrapper,
+  ModelSelectorContainer,
+  ModelComparisonsContainer,
+} from './CompareModels.style';
+
+export function CompareModels() {
+  // NOTE: The actual website doesn't handle CORS requests so we have to hit the S3 buckets for now.
+  const [leftUrl, setLeftUrl] = useState(
+    'http://covidactnow.org.s3-website-us-west-1.amazonaws.com/data/',
```

Comment:
could we make this equivalent to the current site instead, regardless of what domain it is?

Reply:
(i understand this may add a challenge since staging and prod are referring buckets, not the domain itself)

Reply:
Sure, we can make it whatever we want.  Just using `/data/` as the URL should work to point at the current site...  But I'm not sure if that ends up making sense.  Right now left always points to master (by default) and right always points to staging (by default).  If we make left point at the current site instead, that means:
* when you go to https://staging.covidactnow.org/compare/ you will be comparing the staging models against the staging models.
* when you go to https://covidactnow.org/compare/ you will be comparing the prod models against the staging models.

So the staging version ends up being a no-op by default, which seems unhelpful and confusion.

Reply:
@dopry updated this so that "right" always points to `/data/ (thanks!), so on prod it'll always use prod as the left (baseline) and the right will be your local environment.  But you can also run a local http server (e.g. from covid-data-model/) and point at that, so you don't have to mess with running covid-projections locally or copying model files over repeatedly.